src/components/common/Menu.js

```diff
@@ -1,9 +1,19 @@
-import React from 'react'
+import React, {useState, useEffect} from 'react'
 import Tooltip from '@material-ui/core/Tooltip';
 import HelpIcon from '@material-ui/icons/Help';
 import './Menu.scss'
 
-const Menu = ({ title = "", description=[], onChange=(e) => console.log(e), entries = [] }) => {
+const Menu = ({ title = "", description=[], onChange=(event) => console.log(event.target.value), entries = [], reset, updateReset}) => {
+    const [value, setValue] = useState("default");
+
+    const handleChange = e =>{
+        if(reset === true){
+            updateReset();
+        }
+        setValue(e.target.value) 
+         
```

Comment:
At the moment reset seems to work great! However, it seems that the menu itself is broken now, meaning it doesn't do anything when changing because the onChange method is not called. 

I think we need to call  `onChange(e);` here and that should do the trick!

Reply:
Looks good!